ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt

```diff
@@ -0,0 +1,148 @@
+# The Tower Of Babil
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+
+#~~~~~~~~~~#
+# BARNABAS #
+#~~~~~~~~~~#
+
+# -ii 6076 61DE
+
+0.0 "--sync--" sync / 00:0839::Magitek Servicing will be sealed off/ window 0,1
+8.9 "Ground and Pound" sync / 1[56]:[^:]*:Barnabas:6247:/ window 8.9,5
+27.3 "Ground and Pound" sync / 1[56]:[^:]*:Barnabas:62EA:/
+
+40.5 "Dynamic Scrapline" sync / 1[56]:[^:]*:Barnabas:62F0:/ window 40.5,10
```

Comment:
Is there only one id for the negative/positive here?

Reply:
No, good catch. I skipped over that when generating the initial log and it didn't show up again.